src/LibraryManager/CacheService/CacheService.cs

```diff
@@ -26,6 +27,24 @@ public CacheService(IWebRequestHandler requestHandler)
             _requestHandler = requestHandler;
         }
 
+        /// <summary>
+        /// Defines the root cache directory.  This should be consistent across all LibMan tools. 
+        /// </summary>
+        public static string CacheFolder
+        {
+            get
+            {
+                string envVar = "%HOME%";
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    envVar = "%USERPROFILE%";
+                }
+
+                return Path.Combine(Environment.ExpandEnvironmentVariables(envVar), ".librarymanager", "cache");
```

Comment:
Yes.  I don't see a good way around this without risking deleting files that are not part of the cache.  We don't have a good way to determine when the cache is being transitioned to the subdirectory vs. files that are in the .librarymanager folder for some other reason.  That's actually the whole reason for moving the cache root, because `libman cache clean` was deleting the settings config file.